CHANGELOG.md

```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v5.8.0](https://github.com/buildkite/elastic-ci-stack-for-aws/tree/v5.8.0) (2022-02-28)
+[Full Changelog](https://github.com/buildkite/elastic-ci-stack-for-aws/compare/v5.7.2...v5.8.0)
+
+### Added
+- Allow Docker address pool customisation [#968](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/968) (@ouranos)
```

Comment:
This one ended up being more like:

> Smaller docker networks allow 16x as many docker-compose networks

We removed the customisation aspect mentioned in the PR title.